### ISSUE TITLE
An optional sorting order for pop-up reference list 

### ISSUE THREAD

**Issue by lemywong** [hans/obsidian-citation-plugin#66]
When frequently taking notes from a paper, the citation plugin should frequently search the key to insert it, this inconvenience enlightens me that, if the reference list could be ordered by the 'latest used' order, then it would be more convenient. 

To put it further, maybe an optional sorting order would be more preferable,  that it, other than 'latest used', the list could be ordered by 'frequently used', 'latest added', etc.

**Comment by bensleveritt** [comment]
I've found it's in bib-file-order, which for me is oldest to newest. I tend to want the most recent ones, so I'd prefer a sort of newest to oldest, but other options would be nice

**Comment by bensleveritt** [comment]
Actually, looking at the bib file, there's no sure way of knowing which are new, so shy of caching and diffing, there's no way for Citation to show latest added..

**Comment by Klemet** [comment]
I agree that different sorting options for the pop-up reference list would help to access references quicker. I'd especially find a "last used citations" to be most useful.

Hard to know how to do that if everything is currently linked to the `.bib` file. Maybe Better BibTex can export other properties for the entries in the `.bib` file ?
